### PR TITLE
branding for aeqt

### PR DESCRIPTION
**Pull request by Rohithone**
https://www.pivotaltracker.com/n/projects/2481183/stories/184353322#

### CODE REVIEW

**Review by rahulgudim — DISMISSED** [review]

**Review by nisanthyaganti9 — APPROVED** [review]

**Review by rahulgudim — APPROVED** [review]

File: clients/html/src/app/employer-details/employer-details.component.html [hunk 1]
```diff
@@ -490,20 +490,24 @@ <h6 class="sub-heading-text">Dependent {{ j + 1 }}</h6>
             <!-- starts employee fields -->
             <fieldset class="household mb-3">
               <legend>
+                <h1 class="wave_hidden">Employee</h1>
+                <h2 class="wave_hidden">Employee</h2>
+                <h3 class="wave_hidden">Employee</h3>
+                <h4 class="wave_hidden">Employee</h4>
```
Comment (nisanthyaganti9) [inline clients/html/src/app/employer-details/employer-details.component.html:496]: Do we need this hidden tags?

File: clients/html/src/app/employer-details/employer-details.component.html [hunk 2]
```diff
@@ -137,7 +137,7 @@ <h2 class="heading-text">Employee Roster</h2>
             Note: You may use this complete roster download when you apply.
           </p>
           <button type="button" class="btn btn-link">
-            <a href="/get-a-quote/assets/roster_upload_template.xlsx" download
+            <a href="/assets/roster_upload_template.xlsx" download
```
Comment (nisanthyaganti9) [inline clients/html/src/app/employer-details/employer-details.component.html:140]: Did you tested download locally with this change?

**Review by nisanthyaganti9 — CHANGES_REQUESTED** [review]

File: clients/html/src/app/employer-details/employer-details.component.html (hunk 1 above)
Comment (Rohithone) [reply clients/html/src/app/employer-details/employer-details.component.html:496]: Yes, these are recommendations from the wave tool. We should follow these.

File: clients/html/src/app/employer-details/employer-details.component.html (hunk 2 above)
Comment (Rohithone) [reply clients/html/src/app/employer-details/employer-details.component.html:140]: Yes, I did a test in my local, and it's downloading.

**Pull request merged**